Documentation/PackedAssetBuilds.md

```diff
@@ -1,64 +1,121 @@
+
+This guide uses `C:\git\o3de` is the source for the O3DE, cloned from GitHub.
+`C:\git\o3de-multiplayersample` is the location of the clone project. These paths are optional. If you choose to use different paths, amend the command scripts in the rest of the guide accordingly.
+
 ## Windows Profile PAK Setup
 
-### Install WWise
-Go to https://www.audiokinetic.com/download/, create a login, log in, and download the installer. WWise is needed to process and package audio assets in the project.
+Multiplayer Sample uses Wwise gem and assets for audio effects. O3DE engine (and O3DE installers) do not include Wwise support by default. In order to add the Wwise support in the engine, one must first install Wwise SDK and then re-build O3DE engine from source.
+
+### Install Wwise
+Go to https://www.audiokinetic.com/download/, create a login, log in, and download the installer. Wwise is needed to process and package audio assets in the project.
 
-![WWise installer options](Media/wwise_installer_options.png)
+![Wwise installer options](Media/Wwise_installer_options.png)
 
 Inside the installer select the version to use.  Install version **2021.1.11.7933**, select both Authoring and SDK, Microsoft platform.
 
-![WWise version selection](Media/wwise_installer_version_selection.png)
+![Wwise version selection](Media/Wwise_installer_version_selection.png)
 
 > REBOOT (or logout / login). Otherwise, the environment settings won't get picked up for any builds in Visual Studio. They will only apply to command-line builds, and only for any command-line windows that have been opened after the installer finishes.
 
-### Build profile build and process assets
+### Build o3de-multiplayersample and the engine (non-monolithic)
+
+Build a regular profile build of the game as per the [README.md](../README.md) in an engine-centric way. Be sure to build from the source engine and not the installer. Here is an example:
+
+Clone the engine from source.
 
-Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+```shell
+C:\git> git clone https://github.com/o3de/o3de
+```
 
-### Test the profile build
+Navigate to `C:\git\o3de`.
+Create a build folder for non-monolithic build of the engine, for example `C:\git\o3de\build\non_mono`.
 
-* Open the game in editor
-    * load `NewStarBase` level
-    * Verify that game can launch and connect to local server from editor
-* Validate local game launcher can connect to local server
+> The location and the name of the build folder is optional.
+
+Configure the engine in a non-monolithic mode with o3de-multiplayersample project.
+
+```shell
+C:\git\o3de> cmake -S . -B build\non_mono -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
+```
+
+> `-DLY_MONOLITHIC_GAME=OFF` is the default value but for clarity it's specified here explicitly.
+
+Build the Editor. This will compile the project and necessary gems to produce the required game assets.
```

Comment:
If you choose to include the AssetBundler in this build command, you could say `Build the Editor and AssetBundler`

Reply:
Done